cmd/treedb_vector_partition_bench/local_hnsw_attribution_history_test.go

```
@@ -0,0 +1,109 @@
+	report.TopologyIdentitySHA256 = localHNSWAttributionHistoricalTopologySHA256V1[0]
+	if evidence, err := localHNSWAttributionHistoricalBaselineV1(cfg); err != nil || evidence[0].Probe2.Probes != 2 || evidence[0].Probe16.Probes != 16 {
+		t.Fatalf("evidence=%+v err=%v", evidence, err)
+	}
+	for _, tc := range []struct {
+		name string
+		raw  func() []byte
+	}{
+		{"identity", func() []byte {
+			x := cloneLocalHNSWAttributionHistoryReportV1(t, report)
+			x.TopK = 9
+			raw, _ := json.Marshal(x)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `time.Time` found

(errchkjson)